schedview/plot/nightreport.py

```diff
@@ -0,0 +1,233 @@
+import urllib.parse
+from io import StringIO
+
+from astropy.time import Time
+
+from schedview.dayobs import DayObs
+
+
+def night_report_markdown(night_report: dict, heading_level=1) -> str:
+    """Format a night report using Markdown
+
+    Parameters
+    ----------
+    night_report : `dict`
+        A night report, as retured as an element of the list returned by
+        `schedview.collect.nightreport.get_night_report`.
+    heading_level : `int`, optional
+        The markdown heading level for the highest level heading
+        to use, by default 1
+
+    Returns
+    -------
+    night_report : `str`
+        The markdown of the report.
+    """
+    headhash = "#" * heading_level
+
+    report = StringIO()
+    print(f"{headhash} Night report", file=report, end="")
+    if "telescope" in night_report:
+        print(f" for the {night_report['telescope']} telescope", file=report, end="")
+    if "day_obs" in night_report:
+        day_obs = DayObs.from_date(night_report["day_obs"], int_format="yyyymmdd")
+        print(f" on {day_obs}", file=report, end="")
+    print("", file=report)
+
+    if "observers_crew" in night_report:
+        print(f"{headhash}# Observers", file=report)
+        print(", ".join(night_report["observers_crew"]), file=report)
+        print("", file=report)
+
+    if "confluence_url" in night_report and len(night_report["confluence_url"]) > 0:
+        night_plan_block = (
+            "BLOCK" + urllib.parse.urlparse(night_report["confluence_url"]).fragment.split("BLOCK")[-1]
+        )
+        if night_plan_block == "BLOCK":
+            night_plan_block = night_report["confluence_url"]
+        print(f"{headhash}# Night plan", file=report)
+        print(f"[{night_plan_block}]({night_report['confluence_url']})", file=report)
```

Comment:
You might want to make these actual clickable URLs - 
something like 
``` 
display(Markdown(f'Night plan : <a href="{url}" target="_blank" rel="noreferrer noopener">{night_plan_block}</a>'))
```
(translated to print into your stringIO) could  work. 
.. I've since found that if these URLs point to zephyr scale, they work, but otherwise something in TimesSquare or the USDF RSP is trapping them internally (I think there may be supposed to be a proxy to JIRA that hijacks the URL .. JIRA URLs  or github URLs don't work, but zephyr scale ones do) - but it's worth a try, and maybe we can figure out what's stopping them from opening properly.

Reply:
It is already a clickable URL: note the square braces and parenthesis, which is markdown for making a link.
But, it doesn't have your nice `target="_blank" rel="noreferrer noopener"` params, which is an improvement, so I just checked in the change.